exercises/all-your-base/all_your_base_test.rb

```diff
@@ -0,0 +1,114 @@
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'all_your_base'
+
+class AllYourBaseTest < Minitest::Test
+  def test_single_bit_to_one_decimal
+    assert_equal [1], BaseConverter.new.convert(2, [1], 10)
+  end
+
+  def test_binary_to_single_decimal
+    skip
+    assert_equal [5], BaseConverter.new.convert(2, [1, 0, 1], 10)
+  end
+
+  def test_single_decimal_to_binary
+    skip
+    assert_equal [1, 0, 1], BaseConverter.new.convert(10, [5], 2)
```

Comment:
I'd also like to see useful error messages, this will show me something like 
```
Expected: [1,0,1] 
Actual: [5]
```
Which doesn't help me work out what the problem is.
adding a 3rd `message` argument to `assert_equal` lets you give that information as part of the failure message
for example:
`input base: 10, digits: [5], output_base: 2. Expected [1,0,1] but got [5]`